ssh/tailssh/tailssh_integration_test.go

```diff
@@ -131,6 +132,20 @@ func TestIntegrationSSH(t *testing.T) {
 			skip:            os.Getenv("SKIP_FILE_OPS") == "1" || !fallbackToSUAvailable(),
 			forceV1Behavior: false,
 		},
+		{
+			cmd:             `echo "${GIT_ENV_VAR:-unset1} ${EXACT_MATCH:-unset2} ${TESTING:-unset3} ${NOT_ALLOWED:-unset4}"`,
+			want:            []string{"working1 working2 working3 unset4"},
+			skip:            os.Getenv("SKIP_FILE_OPS") == "1" || !fallbackToSUAvailable(),
```

Comment:
I don't think you need to worry about `SKIP_FILE_OPS` here, since you're not accessing any files from the test.

Also, it would be good to make this new functionality work even if `/bin/su` isn't available (i.e. `handleSSHInProcess()`.